pygem/setup/config.py

```diff
@@ -7,42 +7,336 @@
 """
 import os
 import shutil
-import sys
 import yaml
+import argparse
+import ruamel.yaml
```

Comment:
We no longer use `yaml` and `argparse`

```suggestion
import ruamel.yaml
```